components/navbar/Navbar.js

```diff
@@ -165,19 +165,14 @@ export default function Navbar() {
                   item={{ name: `v${app.version}`, url: "/roadmap" }}
                   setIsOpen={setIsOpen}
                 />
-                <div className="relative">
-                  <Link
-                    href="https://github.com/EddieHubCommunity/LinkFree"
-                    aria-current="page"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
+                <Link href="https://github.com/EddieHubCommunity/LinkFree" target="_blank" rel="noreferrer" aria-current="page">
```

Comment:
The formatting isn't quite correct here, this should be broken on to multiple lines as it was before, otherwise when someone else works on the file and saves it, Prettier will reformat the file correctly and it will make their PR seem much bigger and cause confusion